apps/src/templates/HideToolbarHelper.jsx

```diff
@@ -56,44 +46,50 @@ export default class HideToolbarHelper extends React.Component {
   }
 
   isToolbarShowing() {
-    // window.innerHeight is smaller than document.body.offsetHeight when
-    // the iOS 13 Safari toolbar is showing.  It becomes equal when the toolbar
-    // is hidden.  (Interestingly, document.documentElement.clientHeight is
-    // the same as document.body.offsetHeight in both cases.)
-    return window.innerHeight < document.body.offsetHeight;
+    // In general, window.innerHeight is smaller than document.body.offsetHeight
+    // and document.documentElement.clientHeight when the iOS 13 Safari toolbar
+    // is showing. However, as the page is loading, these three values change at
+    // different times when the viewport is loading. Therefore we check the
+    // innerHeight against the offsetHeight and the clientHeight. When the
+    // toolbar is hidden, all values are the same.
+    return (
+      window.innerHeight < document.body.offsetHeight ||
+      window.innerHeight < document.documentElement.clientHeight
+    );
```

Comment:
Very interesting!

Reply:
This could well explain the wider than expected variety of `hid-X` values I'm in seeing in GA.